.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## Description
+
+<!--
+	This PR implements `SAMPLE_NAME` in `SAMPLE_CATEGORY` category.
```

Comment:
I just noticed that these are tabs, not spaces. Did you mean to do that? There isn't anything wrong with tabs per se, but I don't think we use them anywhere else. I just want to check that that was intentional.

Reply:
Good point. I should've set the text editor to `Indent use space`. Lemme redo the indentation.